<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminWindow</class>
 <widget class="QMainWindow" name="AdminWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>614</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stack">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>421</width>
      <height>341</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="page_4">
     <widget class="QLineEdit" name="pass_txt_box">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>160</y>
        <width>201</width>
        <height>20</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLabel" name="pass_label">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>160</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QLabel" name="id_lbl">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>110</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="id_txt_box">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>110</y>
        <width>201</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="loginButton">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>270</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="addButton">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>80</y>
        <width>221</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Driver</string>
      </property>
     </widget>
     <widget class="QPushButton" name="viewAllButton">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>160</y>
        <width>221</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>View History</string>
      </property>
     </widget>
     <widget class="QPushButton" name="logout_btn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>logout</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>42</x>
        <y>62</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Trips</string>
      </property>
     </widget>
     <widget class="QListView" name="listView">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>331</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QWidget" name="formLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>291</width>
        <height>293</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="AddLayout">
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton_4">
         <property name="text">
          <string>Back</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>ID</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEdit_2"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Salary</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="lineEdit_3"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Plate Number</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="lineEdit_4"/>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="lineEdit_5"/>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="lineEdit_6"/>
       </item>
       <item row="7" column="1">
        <widget class="QLineEdit" name="lineEdit_7"/>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Year</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>ADD</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Color</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Model</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>614</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
